### ISSUE TITLE
Ballblazer Champions (Still) Crashing on Android

### ISSUE THREAD

**Issue by aorin1** [notaz/pcsx_rearmed#312]
As of r231 bdee1a build, the game runs fine on Windows but still crashes on Android.
Tried a new USA dump to make sure it wasn't a bad copy.

Android 6.0.1 arm7a

**Comment by notaz** [comment]
Crashes at what point? 32bit or 64bit?

For me it gets ingame on a rpi4.

**Comment by aorin1** [comment]
This one crashes almost immediately post boot. It's very unstable and it never reached in-game for me. I've tried even the Japanese or PAL dump from Resumo and it crashed all the same, on Android.

**Comment by aorin1** [comment]
Strange, it 

> For me it gets ingame on a rpi4.

Is there any other parameter that it could cause the crashes specifically on Android?
Like I mentioned, the game works fine on Windows and I've updated PCSX-R on both Win and Android.

The same dump was tested, along with the Japanese, and since all other games work fine on Android as well, it's possible it's not something related to the Core or other Settings.

**Comment by aorin1** [comment]
> No idea. It's not like I'm putting in game crashing options intentionally.

I believe the Play Store version is way outdated, yes, and I sideload the most updated versions to update it from time to time. That's what I just did, sideloaded the latest APK and it updated OK, tried running the game and it crashed within 2 to 3 seconds. Then it occurred that instead of trying to upload the core within the Core Downloader, I've downloaded the latest PCSX-R core to the desktop and copied it to the SD card, using the file explorer with root privileges I moved it to the "core" folder, replacing the older core manually, it didn't seem to have any difference in size but fine, then bang, the game boots and goes into gameplay.

5 Days ago you updated the original open issue regarding this [https://github.com/libretro/pcsx_rearmed/commit/dc6c7f249ff17e808a5e740d7cfc43dfa112a92e](url) and I've tried updating the core at least two to three times in the following days, so here's my doubt, was it not updating for some reason from inside RA?

In any case, thanks again!
